packages/ggit/src/output/output.ts

```diff
@@ -48,63 +48,41 @@ export interface OutputFile<M> {
   Day: Daycent<M>;
 }
 
-export type ErrorResponse = InputValidationError | AllErrors;
-
-/**
- *
- * @example
- */
-export interface InputValidationError {
-  InputErrors: InputErrors;
-}
-
 /**
  *
  * @example
  */
-export interface AllErrors {
-  AllErrors: {
-    Errors: Error;
+export interface ErrorResponse {
+  AllErrors?: {
+    Errors: {
+      ModelRun: ModelRunErrorCollection | ModelRunErrorCollection[];
+    };
+  };
+  InputErrors?: {
+    InputValidationErrors?: {
+      ModelRun: ModelRunErrorCollection | ModelRunErrorCollection[];
+    };
+    APIBuildErrors?: {
+      BuildError: ModelRunErrorCollection | ModelRunErrorCollection[];
+    };
   };
 }
 
 /**
  *
  * @example
  */
-export interface Errors {
-  AllErrors: InputErrors;
-}
-/**
- *
- * @example
- */
-interface InputErrors {
-  InputValidationErrors: Error;
-}
-
-/**
- *
- * @example
- */
-interface Error {
-  ModelRun: ModelRunErrors;
-}
-
-/**
- *
- * @example
- */
-interface ModelRunErrors {
+export interface ModelRunErrorCollection {
   '@name': string;
-  Error: Error | Error[];
+  '#text'?: string;
+  Error?: ModelRunError | ModelRunError[];
```

Comment:
do you know why these can be undefined? seems weird to have a model run error without any error or text

Reply:
Some of the errors we receive have an error defined, some have text defined, & I haven't been able to get my hands on docs that say what scenarios can happen when ... 😬 

Ex, here's an example where we have an input error that doesn't have an error included. There are also plenty that don't have text set, but have an error object
```
{
	"InputErrors":{
		"APIBuildErrors":{
			"BuildError":{
				"@name":"externalId=XXXXXX",
				"#text":"Location externalId=XXXXX contains soil map units that we cannot model..."
			}
		}
	}
}
```
We could probably assume that either text or error is sent, but since I'm not sure I've written the types that everything is optional to encourage us to guard against missing fields when we consume this

Reply:
makes sense! Keeping it lean for now sounds good to me-- we can add the type guards later and then separate the two interfaces accordingly if/when we understand it better.